templates/focus-ios.xliff

```diff
@@ -729,6 +749,74 @@
         <source>URL to open</source>
         <note>Label for input to set a favorite URL to be opened by Siri.</note>
       </trans-unit>
+      <trans-unit id="Tip.Biometric.Title">
+        <source>Lock %@ when a site is open:</source>
+        <note>Text for a label that indicates the title for biometric tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricFaceId.Description">
+        <source>Turn on Face ID</source>
+        <note>Text for a label that indicates the description for biometric Face ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.BiometricTouchId.Description">
+        <source>Turn on Touch ID</source>
+        <note>Text for a label that indicates the description for biometric Touch ID tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Description">
+        <source>Read more about this and other updates to %@.</source>
+        <note>Text for a label that indicates the description for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Release.Title">
+        <source>Why yes, we do have a fresh new look!</source>
+        <note>Text for a label that indicates the title for release tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Description">
+        <source>Page Actions &gt; Request Desktop Site</source>
+        <note>Text for a label that indicates the description for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.RequestDesktop.Title">
+        <source>Want to see the full desktop version of a site?</source>
+        <note>Text for a label that indicates the title for request desktop tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.ShareTrackers.Description">
+        <source>%@ trackers blocked so far</source>
+        <note>Text for a label that indicates the description for share trackers tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.ShareTrackers.Title">
+        <source>You browse. %@ blocks.</source>
+        <note>Text for a label that indicates the title for share trackers tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Shortcuts.Description">
+        <source>Select Add to Shortcuts from the %@ menu</source>
+        <note>Text for a label that indicates the description for shortcuts tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.Shortcuts.Title">
+        <source>Create shortcuts to the sites you visit most:</source>
+        <note>Text for a label that indicates the title for shortcuts tip.</note>
+      </trans-unit>
+      <trans-unit id="Tip.SiriErase.Description">
+        <source>Add Siri shortcut</source>
+        <note>Text for a label that indicates the description for siri erase tip.</note>
```

Comment:
comment is a bit confusing because it only aludes to siri erase tip, and not about the Shortcut section of the string. Would be good to explain how those are related